Python/71_Trust_But_Verify.ipynb

```diff
@@ -532,8 +532,8 @@
     "        with tempfile.TemporaryDirectory() as tmpdirname:\n",
     "            if platform.system() == 'Windows':\n",
     "                for i, fname in enumerate(image_file_name):\n",
-    "                    shutil.copy(os.path.abspath(fname),\n",
-    "                               os.path.join(tmpdirname,str(i)))\n",
+    "                    os.symlink(os.path.abspath(fname),\n",
+    "                               os.path.join(tmpdirname,str(i)))  \n",
     "            else:\n",
```

Comment:
Why was the shutil.copy changed to os.symlink? On windows 10 for a standard user to create symbolic links requires an update to the OS settings or admin privileges (see os.symlink [documentation](https://docs.python.org/3/library/os.html)).

Reply:
This was a workaround for an hour I was getting in Windows, but I can't seem to remember what the error was at the moment.  I'll go back and retest this.

Reply:
This seems to work fine with shutil.copy.  Not sure what the original error was. Thanks for pointing this out.